pkg/config/analysis/msg/messages.yaml

```diff
@@ -625,3 +625,14 @@ messages:
         type: string
       - name: webhook
         type: string
+
+  - name: "ReferencedInternalGateway"
```

Comment:
Could you also add this to the https://istio.io/latest/docs/reference/config/analysis/ page?

Reply:
BTW I think we need a release note for this.